numpy/testing/_private/parameterized.py

```diff
@@ -45,29 +45,7 @@
 
 from unittest import TestCase
 
-PY2 = sys.version_info[0] == 2
-
-
-if PY2:
-    from types import InstanceType
-    lzip = zip
-    text_type = unicode
-    bytes_type = str
-    string_types = basestring,
-    def make_method(func, instance, type):
-        return MethodType(func, instance, type)
-else:
-    # Python 3 doesn't have an InstanceType, so just use a dummy type.
-    class InstanceType():
-        pass
-    lzip = lambda *a: list(zip(*a))
-    text_type = str
-    string_types = str,
-    bytes_type = bytes
-    def make_method(func, instance, type):
-        if instance is None:
-            return func
-        return MethodType(func, instance)
+lzip = lambda *a: list(zip(*a))
```

Comment:
I'd probably inline this one too

Reply:
Yep, don't know why I didn't in the first pass.